### PR TITLE
Edition and WKT export

### PR DESCRIPTION
**Pull request by maltaesousa**

### CODE REVIEW

**Comment by maltaesousa** [comment]
@kalbermattenm 
You can test it here:
https://sitn.ne.ch/web/sitnlayers/

File: index.html [hunk 1]
```diff
@@ -1,31 +1,90 @@
...
-    <title>sitnLayers</title>
-  </head>
-  <body>
-    <h2>Exemple 1</h2>
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="assets/ol/ol.css" type="text/css">
+  <link rel="stylesheet" href="assets/ol-ext/ol-ext.min.css" type="text/css">
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
```
Comment (kalbermattenm) [inline index.html:11]: Maybe it would be good to locally download all the needed packages

File: index.html [hunk 2]
```diff
@@ -1,31 +1,90 @@
...
+        buttons: [
+          'edit',
+          'createPoint',
+          'createLineString',
+          'createPolygon',
+          'save'
+        ],
+        saveFormats: ['WKT'],
+        baseLayers: [
+          '1-plan_ville',
```
Comment (kalbermattenm) [inline index.html]:
```suggestion
          'plan_ville',
```

File: index.html [hunk 3]
```diff
@@ -1,31 +1,90 @@
...
+          '3-plan_cadastral',
```
Comment (kalbermattenm) [inline index.html]:
```suggestion
          'plan_cadastral',
```

File: index.html [hunk 4]
```diff
@@ -1,31 +1,90 @@
...
+          'ortho2017'
```
Comment (kalbermattenm) [inline index.html]:
```suggestion
          'ortho'
```

File: js/sitnlayers.js [hunk 5]
```diff
@@ -44,32 +43,207 @@
...
+              else console.log(features.getLength() + ' object(s) deleted.')
+              for (var i = 0, f; (f = features.item(i)); i++) {
+                _vectorLayer.getSource().removeFeature(f)
+              }
+              selectCtrl.getInteraction().getFeatures().clear()
+            }
+          }))
+
+          var selectCtrl = new ol.control.Toggle({
+            html: '🖢',
```
Comment (kalbermattenm) [inline js/sitnlayers.js]: Something went wrong here (see my snapshot)

**Review by kalbermattenm — CHANGES_REQUESTED** [review]

**Comment by kalbermattenm** [comment]
It is really cool, just some issue with some button images:

image

**Comment by kalbermattenm** [comment]
Maybe we should also quickly talk about the whole thing and the UI experience with @mriedo (tomorrow...)

**Comment by kalbermattenm** [comment]
It looks good.

Maybe you could get rid of the "Save" button and create an external button "Enregistrer" (like the "Charger" button).

Finally, will the WMTS layer tree be part of another PR ?

File: index.html (hunk 1 above)
Comment (maltaesousa) [reply index.html:11]: I agree! But bootstrap is only required for the demo, it is not used in the plugin.

**Pull request merged**